taca/illumina/Runs.py

```diff
@@ -276,6 +280,15 @@ def transfer_run(self, t_file, analysis):
                                        prefix="", log_dir=self.run_dir)
         except subprocess.CalledProcessError as exception:
             os.remove(os.path.join(self.run_dir, 'transferring'))
+            #Send an email notifying that the transfer failed
+            runname =os.path.basename(os.path.normpath(self.run_dir))
```

Comment:
well you have this it is in `self.id` because here you are in the object Run as there fore you have access too all functions and variables of this object.